### DISCUSSION TITLE
VSCode cannot connect to the Extensions Marketplace

### DISCUSSION THREAD

**Discussion by longqua69** [EddieHubCommunity/support#1060] [Q&A] [answered]
Hello folks,

I'm facing an issue with VSCode.

![vscode](https://user-images.githubusercontent.com/37626565/110354517-d5653300-806a-11eb-8b7a-90f132d4ae32.png)

I've tried some ways were discussed that still didn't work:

1. https://github.com/microsoft/vscode/issues/115818
2. https://github.com/microsoft/vscode/issues/97484

In addition, I disabled all VSCode extensions, the result is below:

![Disabled](https://user-images.githubusercontent.com/37626565/110355636-0c881400-806c-11eb-8f99-49615ce90f5f.PNG)

Hope this discussion can help me fix the issue and anyone else in the future. 😃 

Thanks & Best regards,

Anh Tran

**Comment by eddiejaoude** [comment, marked as answer]
Did you fix it?

I would uninstall and reinstall it.

**Reply by longqua69** [reply]
Sorry for lateness, I missed this topic.
So far, I haven't figured the root cause of this error. However, I don't face this error anymore.
My networking knowledge is quite poor. Maybe I did something unintentionally and it works. I can't explain it.

**Reply by eddiejaoude** [reply]
No worries, glad you got it sorted 🤓
